_posts/2022/2022-01-31-designer-experience-2022.md

```diff
@@ -0,0 +1,59 @@
+
+#### Work Mode: ON
+
+We kickstarted our work as SDSLabs members with our Minors, where we worked on minor fixes and issues in the ongoing projects under the guidance of our mentors. I worked on identifying the UI/UX issues on Accounts, the authentication system of SDSLabs, and with that, I learned the importance of documenting your work. 
+
+Then we had our Majors, where our scope of work and learning increased. We were assigned projects based on our interests as well as our performance in Minors, and I got to work on Backdoor, the CTF platform of SDSLabs.
+
+We designed some nice posters, and I also tried my hands on Adobe After Effects while working on the release video of our project Study Portal. Apart from the excitement of learning something new, the sheer joy of getting a ++ on slack (our internal motivation system in action XD) is unmatched. 
+
+There’s always happening a lot out there in the tech and design world, so how can the environment at SDSLabs be possible unaffected? We would know about some events or hackathons being organized, and with the slightest possibility of finding an opportunity to learn or build something cool (and often winning some awesome prizes too!), we would have a meeting, divide ourselves into teams, and just get going. 
+
+Not to forget that internal hackathons, sprints, and lectures are a regular practice at SDSLabs
+
+#### Fun Mode: They forgot to design an OFF button for that
+
+We indeed had a lot of fun online with meets turning into ‘bakar sessions’ and a few chapos. Even online Farewell for our seniors was as grand as it could possibly be made virtually. But the real fun began when we reached campus. 
+
+With many more chapos (for successful Minors, Majors, any small or big achievement or for no good-reason-at-all) being given either at the best eating outlets at Roorkee where we literally filled an entire hall or stacks of Pizza boxes and cold drinks being delivered at the Office, the ambiance at SDSLabs is always perfect for learning from your peers and seniors, coming up with crazy ideas and implementing them, just chilling while sitting on the bean bags or going on random late-night walks if you feel low.
+
+A lot of us spent most of our time at SDSLabs, so whenever any of my friends saw me leaving my hostel room with my laptop bag strapped on my shoulders, they almost always knew where I was headed to!
+
+
+## Key takeaway
+
+To sum it up in one line, my journey at SDSLabs so far has been better than what I could have ever expected. With seniors playing a great role as mentors guiding me not only in the professional sphere but helping me deal with the ups and downs of life as well, and my peers being just the right like-minded people to have conversations and discuss ideas with, SDSLabs has helped me grow.
+
+Recruitments will be held soon. If you have the slightest interest in anything I have talked about so far, do not miss the chance!
```

Comment:
can add: Or you can always just by SDSLabs to find some of the most amazing people in IITR in their natural habitat :P (no need to incorporate the entire line as is, but you get the point right)